espree.js

```diff
@@ -517,7 +517,18 @@ function scanPunctuator() {
         }
     }
 
-    // 3-character punctuators: === !== >>> <<= >>=
+    // 3-character punctuators: ... === !== >>> <<= >>=
```

Comment:
Double check this - pretty sure this code is already somewhere in there because it's used for JSX. Look for ecmaFeatures.jsx